src/main/java/com/academia/library/validator/OrderValidator.java

```diff
@@ -0,0 +1,65 @@
+package com.academia.library.validator;
+
+import com.academia.library.dto.OrderDetailsRequest;
+import com.academia.library.dto.OrderDetailsRequest.OrderDetailRequestDto;
+import com.academia.library.dto.OrderRequest;
+import com.academia.library.exception.BookNotFoundException;
+import com.academia.library.exception.OrderStatusException;
+import com.academia.library.model.Order;
+import com.academia.library.model.OrderStatus;
+import com.academia.library.repository.BookRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Component
+@RequiredArgsConstructor
+public class OrderValidator {
```

Comment:
Please try to rename methods in this class for clear understanding

Reply:
Ok, thank